Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ruby:2.7.1-alpine3.11
+
+RUN apk update \
+    && apk upgrade \
```

Comment:
we have a convention (across all languages) to do `&&` and `||` at the end of th line, not the beginning of the next one, for example
```
RUN apk update && \
     apk upgrade && \
    ...
```